pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -447,6 +447,26 @@ func (c *Client) CreateNamespace(namespace, manifest string) error {
 	return err
 }
 
+// GetChildInstances returns all instances that were created as dependencies of a parent instance
+func (c *Client) GetChildInstances(parent *v1beta1.Instance) ([]v1beta1.Instance, error) {
```

Comment:
Should this reside in the `instance_types_helpers.go`?

Reply:
It shouldn't, because it's doing more than extract information from or mutate a `v1beta1.Instance`. With that, the question is rather if `GetInstance` should be in `instance_types_helpers.go` because it's the only function in there that uses a Kubernetes client.